app/views/accounts/_connected_app.html.erb

```diff
@@ -1,4 +1,8 @@
 <div class="padding-1 border-top border-left border-right border-primary-light">
+  <% if flash[:email_updated_identity_id] == identity.id %>
+    <%= render AlertComponent.new(type: :success, id: 'emails', class: 'margin-bottom-2', message: t('account.connected_apps.email_update_success_html', sp_name: identity.display_name).html_safe) %>
```

Comment:
I don't think we'll need an `id` here, and `_html` suffix on the key should take care of the `.html_safe` unescaping ([ref](https://guides.rubyonrails.org/i18n.html#:~:text=Keys%20with%20a%20%27_html%27%20suffix%20and%20keys%20named%20%27html%27%20are%20marked%20as%20HTML%20safe.)).

```suggestion
    <%= render AlertComponent.new(type: :success, class: 'margin-bottom-2', message: t('account.connected_apps.email_update_success_html', sp_name: identity.display_name)) %>
```

Reply:
Thanks, I didn't remember the `_html` naming behavior. Why do you think the `id` is not needed?

Reply:
> Why do you think the `id` is not needed?

I'd answer with another question, being: why **is it** needed?

I'm supposing it's not needed, so we shouldn't have code for it (maintenance burden, future uncertainty on its purpose, potential for conflicts with duplicate IDs, etc etc).

Reply:
Ah, you meant the `id` in the constructor arguments. I was confused and thought that you meant it in the previous line.